README.md

```diff
@@ -103,6 +103,17 @@ export default App;
 
 You can find documentation and examples of all our components in this [storybook](https://components.gnosis-safe.io/).
 
+## Testing
+
+Snapshot tests are generated automatically from the Storybook stories.
```

Comment:
can we add the link to the doc? https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core